tests/test-webapps/test-jetty-webapp/src/main/config/demo-base/modules/demo.mod

```diff
@@ -4,20 +4,27 @@
 # Additional ini files are in demo-base/start.d
 # 
 
-# Enable security via jaas, and configure it
---module=jaas
-jetty.jaas.login.conf=etc/login.conf
+[depends]
+rewrite
+jaas
+test-keystore
+
 
+[xml]
 # Enable rewrite examples
---module=rewrite
 etc/demo-rewrite-rules.xml
 
+# Add the test realm
+etc/test-realm.xml
+
+[ini-template]
+# Enable security via jaas, and configure it
+jetty.jaas.login.conf=etc/login.conf
+
 # Websocket chat examples needs websocket enabled
 # Don't start for all contexts (set to true in test.xml context)
 org.eclipse.jetty.websocket.jsr356=false
---module=websocket
```

Comment:
The `websocket` module was enabled before, but it's not now? Is that right, maybe enabled transitively?

Reply:
it is enabled explicitly for the demo-base in the pom.xml
I removed all the module enables from this file because it goes on the command line without processing any transitive dependencies and thus can actually stop a subsequent --add-to-start